app/__tests__/dashboard.test.ts

```diff
@@ -1,21 +1,4 @@
 import { shortest } from "@antiwork/shortest";
-import { clerk, clerkSetup } from "@clerk/testing/playwright";
-
-let frontendUrl = process.env.SHORTEST_TEST_BASE_URL ?? "http://localhost:3000";
-
-shortest.beforeAll(async ({ page }) => {
-  await clerkSetup({
-    frontendApiUrl: frontendUrl,
-  });
-  await clerk.signIn({
-    page,
-    signInParams: {
-      strategy: "email_code",
-      identifier: "shortest+clerk_test@example.com",
-    },
-  });
-
-  await page.goto(frontendUrl + "/dashboard");
-});
 
+shortest("Login to the App using magic link. Use this email: 'stay-ants@taoyeoeo.mailosaur.net'");
```

Comment:
```suggestion
shortest("Login to the app using a magic link with this email: stay-ants@taoyeoeo.mailosaur.net");
```

Reply:
Changed it to the one you sent me. `mission-health@rdt7stzf.mailosaur.net`

Reply:
Sounds good but please make the other changes, I think it makes it more human-readable / a better example once AI is used to generate new shortest tests for other repos